### PR TITLE
CRDCDH-1445 Uploader CLI Tool Download Options

### PR DESCRIPTION
**Pull request by Alejandro-Vega**
### Overview

Updated existing Uploader CLI Tool dialog to include additional downloads for additional platforms.

### Change Details (Specifics)

- Updated styling of dialog
- Updated description text
- Created Package Table which includes the package type, platform, and download link for source, Windows, and MacOS platforms
- Introduced new env variables `REACT_APP_UPLOADER_CLI_WINDOWS` and `REACT_APP_UPLOADER_CLI_MAC`
- The "Download" column, which includes download links and icon, directly uses the file name from the env variable link
- Added tests for Package Table
- Removed previous single download button

> [!Warning]
> This feature is for the 3.1.0 release, not 3.0.0

> [!Warning]
> The binary distributions for Windows and MacOS have not yet been created by the BE. The download table cell will be blank until the environment variable is populated with the links.

### Related Ticket(s)

[CRDCDH-1445](https://tracker.nci.nih.gov/browse/CRDCDH-1445)

### CODE REVIEW

File: src/components/UploaderToolDialog/PackageTable.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  fontFamily: "'Nunito', 'Rubik', sans-serif",
+  fontWeight: 700,
+  fontSize: "16px",
+  lineHeight: "19.6px",
+  color: "#365F71",
+  textDecoration: "underline",
+  marginRight: "8px",
+});
+
+const PackageTable = () => (
```
Comment (amattu2) [inline src/components/UploaderToolDialog/PackageTable.tsx:63]: When opening the dialog on the Model Navigator page, the spacing within the table gets messed up. Can you take a look at it? 

<details><summary>Normal vs. Model Navigator</summary>
<p>
Screenshot 2024-08-13 at 10 02 40 AM
</p>
<p>
Screenshot 2024-08-13 at 10 02 49 AM
</p>
</details> 

P.S. if model navigator won't load the model, switch `REACT_APP_DEV_TIER` to `DEV`

**Review by amattu2 — CHANGES_REQUESTED** [review]
Really solid implementation, especially with handling for the dynamic filename of the downloads.

I annotated two simple issues, everything else LGTM.

File: .env.example [hunk 2]
```diff
@@ -9,6 +9,8 @@ REACT_APP_DEV_TIER=""
 
 # Uploader CLI Release
 REACT_APP_UPLOADER_CLI=""
+REACT_APP_UPLOADER_CLI_WINDOWS="",
+REACT_APP_UPLOADER_CLI_MAC="",
```
[the remark this answers is not in the document]
Comment (Alejandro-Vega) [reply .env.example]: Thanks! Good catch!

File: src/components/UploaderToolDialog/PackageTable.tsx (hunk 1 above)
Comment (Alejandro-Vega) [reply src/components/UploaderToolDialog/PackageTable.tsx:66]: Ah forgot to check this. Thanks for catching this!

**Review by amattu2 — APPROVED** [review]
LGTM

**Pull request merged**